src/components/Pages/Logout/Logout.jsx

```diff
@@ -0,0 +1,22 @@
+import propTypes from 'prop-types';
+import noop from '../../../shared/noop';
+import { Button } from '../../Buttons/Button/Button';
+
+export function Logout({ logoutHandler, userName, role }) {
+  return (
+    <>
+      <span>{`${userName}: ${role}`} </span>
```

Comment:
create method

Reply:
now I am getting this data via Provider. I think the method looks redundant here.

```suggestion
export function Logout() {
  return (
    <AuthContext.Consumer>
      {({ name, role, logoutHandler }) => {
        function showUser() {
          return <span>{`${name}: ${role} `}</span>;
        }

        return (
          <>
            {showUser()}
            <Button type='button' title='Logout' onClick={logoutHandler} />
          </>
        );
      }}
    </AuthContext.Consumer>
  );
}
```